### ISSUE TITLE
Back in QA2 doesnt work ?

### ISSUE THREAD

**Issue by 0xae** [PusherBkps/rga2.0#1]
When we press the back button in quests of QA2 of an individua who doesnt have 'parcelas', the back button, keeps sending him to the same place. This has to do with this code:
```javascript
            if (!total_de_parcelas) {
                // /parcela/:id_agregado/:id_membro/:id_parcela/:ap/:MembrosID
                setTimeout(function (){
                    const params = {
                        id_qa2 : OBJECT_ID
                    };
                    $state.go("app.tabs.quest_qa2", params);
                },0);
            }
```

It can be found on [js/controllers/typeCtrl.js:35](https://github.com/PusherBkps/rga2.0/blob/master/js/controllers/typeCtrl.js#L35)

**Comment by 0xae** [comment]
Solved by 28294a2ee6f658069eced9c83d377509f5d73f88

**Comment by 0xae** [comment]
Couldnt solve this bug without the help of this [comment](https://github.com/angular-ui/ui-router/issues/92#issuecomment-29160259)
